src/utils/updaters/saveEventToStorage.js

```diff
@@ -0,0 +1,10 @@
+import { getEvents } from "../getters/getEvents.js";
+
+export function saveEventToStorage(event, extEvent, calendarInfo) {
+	const extEvents = calendarInfo.extEvents;
+	extEvents.push(extEvent);
+
+	const events = getEvents();
+	events.push(event);
```

Comment:
It's generally better to manage your state without mutating it.
Instead of pushing to the array and making a mutation on the existing object, you could save an entirely new array as a result:
`localStorage.setItem("events", JSON.stringify([...events, event]));`

Most JS array methods return a new array value see `.filter(), .map()`.